docs/topics/images.md

```diff
@@ -566,3 +566,15 @@ If a user navigates directly to the URL of the SVG file embedded scripts may be
 -   setting `Content-Disposition: attachment` will cause the file to be downloaded rather than being immediately rendered in the browser, meaning scripts will not be executed (note: this will not prevent scripts from running if a user downloads and subsequently opens the SVG file in their browser).
 
 The steps required to set headers for specific responses will vary, depending on how your Wagtail application is deployed.
+
+(heic_heif_images)=
+
+## HEIC / HEIF images
+
+`.heic` images are not widely supported on the web, but may be encountered when exporting images from Apple devices. Wagtail does not allow upload of these by default, but this can be enabled by adding "heic" to `WAGTAILIMAGES_EXTENSIONS`:
+
+```python
+WAGTAILIMAGES_EXTENSIONS = ["gif", "jpg", "jpeg", "png", "webp", "heic"]
+```
```

Comment:
I tried uploading a `.heif` image from https://filesampleshub.com/format/image/heif but it didn't work. Wikipedia says HEIF images can have the following extensions: `.heif, .heifs; .heic, .heics; .avci, .avcs; .HIF`

Do we need to be more explicit in saying that only `.heic` is supported? I drafted a line in my review i.e.

```md
Note that to upload HEIC / HEIF images, the file extension must be `.heic` and not `.heif` or other extensions.
```

Reply:
Yeah, I'm not really clear on how all these names and extensions relate to each other, and whether you're likely to encounter anything other than `.heic` in the wild - or indeed whether those other extensions are likely to be videos, and if so, whether claiming to support them is putting us on the hook for more complexity than we want. For that reason I decided to avoid committing to more than the minimum known use-case - namely, "if you pull photos off your iphone and they turn out to have an .heic extension, you should be able to upload them like any other image".

Happy with your suggested edits - will update and merge now!